app/uk/gov/hmrc/hmrcemailrenderer/templates/onlinepaymentservice/face_to_face_payment_email_successful.scala.html

```diff
@@ -49,34 +41,34 @@
         <th>Date</th>
         <td>@params("dateOfPayment")</td>
     </tr>
-    @for((taxType, amount, transactionFee, transactionNumber) <- itemList) {
+
     <tr>
         <th>Tax</th>
-        <td>@taxType</td>
+        <td>@params("taxType")</td>
     </tr>
     <tr>
         <th>Payment reference</th>
-        <td>@transactionNumber</td>
+        <td>@params("transactionNumber")</td>
     </tr>
-    }
+
     <tr>
         <th>Tax Reference</th>
         <td>@params("transactionReference")</td>
     </tr>
-    @for((taxType, amount, transactionFee, transactionNumber) <- itemList) {
+
     <tr>
         <th>Amount</th>
-        <td>@amount</td>
+        <td>formatAmountInPence(params(amount))</td>
```

Comment:
**formatAmountInPence** will not resolve into a value, also what is the `amount` there?
I don't see it being defined in the template, could you run the service locally and verify the same?